labs-ios-starter/Model Controller/ProfileController.swift

```diff
@@ -52,7 +54,8 @@ class ProfileController {
         var request = URLRequest(url: requestURL)
         
         request.addValue("Bearer \(oktaCredentials.idToken)", forHTTPHeaderField: "Authorization")
-        
+        self.bearer = oktaCredentials.idToken
+        print(bearer)
```

Comment:
We might want to remove this print statement.

Reply:
Print statement has now been removed!